### PR TITLE
Adding serverRender function to client side

### PR DESCRIPTION
**Pull request by Malintha**

### CODE REVIEW

File: apps/social/themes/default/js/lib/caramel.handlebars.client.js [hunk 1]
```diff
@@ -239,6 +242,22 @@
...
     caramel.unloaded = {};
 
+    /**
+     * This Function can be used to render a given template from the server side.
+     * @param areas Partial that needs to be rendered.
+     * @param options Parameters for the AJAX request
+     */
+    caramel.serverRender = function(areas, options) {
+        var headers = options.headers || (options.headers = {});
+        if(!options.hasOwnProperty("dataType")) {
```
Comment (cnapagoda) [inline apps/social/themes/default/js/lib/caramel.handlebars.client.js]: Formatting issue

**Comment by Malintha** [comment]
@cnapagoda formatting issues fixed with 0051c8d0a79bd7d551fdef3a36d5264c63a048cc

File: apps/store/themes/store/partials/top_assets.hbs [hunk 2]
```diff
@@ -12,42 +12,44 @@
...
+            <div class="ast-container " id="ast-container">
+                <div class="container-fluid ">
+                    <div class="title">
+                        {{#if ../assetMeta.q}}
+                            <a class="asset-type-{{rxt.shortName}}" href="{{tenantedUrl "/assets/"}}{{rxt.shortName}}/list?q={{../../assetMeta.q}}">
+                                {{rxt.pluralLabel}}
+                            </a>
+                        {{else}}
+                            <a class="asset-type-{{rxt.shortName}}" href="{{tenantedUrl "/assets/"}}{{rxt.shortName}}/list?">
+                                Recent
```
Comment (cnapagoda) [inline apps/store/themes/store/partials/top_assets.hbs]: Need to use "t" helper here

Comment (Malintha) [inline apps/store/themes/store/partials/top_assets.hbs]: Please find the fix at e11b073ed2126caea55eccfa6ef344aaa0d3c998

**Pull request merged**